File: SpringBootBackend/src/main/java/com/th/dto/ReviewDTO.java
package com.th.dto;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReviewDTO extends BaseDTO {
    private Long bookingId;
    private Long userId;
    private Long vendorId;
    private int rating;
    private String comment;
    private LocalDateTime reviewDate;
}

File: DOTNET BACKEND/TransitHive/Dtos/ReviewDto.cs
namespace TransitHive.Dtos
{
    public class ReviewDto
    {
        public int ReviewId { get; set; }
        public int? BookingId { get; set; }
        public int UserId { get; set; }
        public int? VendorId { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime ReviewDate { get; set; }
    }
}
